### DISCUSSION TITLE
Run qalc in windows console

### DISCUSSION THREAD

**Discussion by Areshkew** [Qalculate/libqalculate#466] [Q&A] [answered]
Hi, I can't use the cli in external consoles, can U help me? :c

**Comment by hanna-kn** [comment, marked as answer]
You need to add c:\Program Files\Qalculate (or wherever Qalculate is installed) to the PATH environment variable (in Advanced system settings -> Environment Variables), or use the full path to the binary qalc.exe (e.g. "c:\program files\qalculate\qalc.exe").

**Reply by Areshkew** [reply]
Thank you :D
